### DISCUSSION TITLE
欢迎使用neno，提出你的看法，在这里进行讨论

### DISCUSSION THREAD

**Discussion by Mran** [openneno/neno#10] [Announcements]
欢迎使用neno，提出你的看法，在这里进行讨论

**Comment by Jones-F** [comment]
Mran，您好！

在 21 年 5 月份已经有人提出[开发 docker 版本](https://github.com/openneno/neno/issues/4)，以进行私有化部署，之后也不断有人提出[这个问题](https://github.com/openneno/neno/issues/12)。

希望您能尽快构建完整后端并推出 docker 版本，非常感谢您的付出！

**Reply by Mran** [reply]
这个项目没有后端,不知道你指的是前端部分进行docker部署吗?

**Reply by Jones-F** [reply]
Mran，五一快乐！

您有打算开发后端的存储数据库吗？

如果具备后端数据库，希望能有离线自部署的 docker 容器，而不是利用 GitHub 进行存储。

**Reply by Mran** [reply]
我明白你的意思了，你是想把数据存到自己的服务器上。可以安排，我想想要怎么做。
